float64[] anglesInRad
float64[] distances
geometry_msgs/Point[] points
time time 
